docs/app/partials/getting-started.tmpl.html

```diff
@@ -1,38 +1,84 @@
 <div ng-controller="GuideCtrl" class="doc-content">
   <md-content>
-    <p><em>New to Angular.js? Before getting into Angular Material, it might be helpful to
-      <a href="https://egghead.io/articles/new-to-angularjs-start-learning-here" target="_blank"
-         title="Link opens in a new window">read about the framework</a>.</em></p>
+    <p><em>New to Angular.js? Before getting into Angular Material, it might be helpful to:</em>
+    <ul>
+      <li>watch the videos about <a href="https://egghead.io/articles/new-to-angularjs-start-learning-here" target="_blank"
+               title="Link opens in a new window">Angular.js framework</a></li>
```

Comment:
Possibly reword: "watch the videos about the Angular.js framework".

Also, why is the title for both of these "Link opens in a new window"? Shouldn't it be related to the link? _(I realize this was not your change, just wondering if there is a reason this was done.)_